### PR TITLE
Feature/add command stubs

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: src/main/java/de/retest/recheck/cli/subcommands/Diff.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package de.retest.recheck.cli.subcommands;
+
+import java.util.Arrays;
+import java.util.List;
+
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+@Command( name = "diff", description = "Show all changes or all changes for a certain state" )
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/cli/subcommands/Diff.java]: Why a different import here than e.g. in `Commit`?

File: src/main/java/de/retest/recheck/cli/subcommands/Stage.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import java.util.Arrays;
+import java.util.List;
+
+//import de.retest.recheck.ChangeApplier;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+@Command( name = "stage", description = "Stage all changes of a certain state and all same changes to other states" )
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/cli/subcommands/Stage.java]: Descriptions should end with consistently, i.e. with or without a period (compare e.g. `description` below).

File: src/main/java/de/retest/recheck/cli/subcommands/Stage.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		System.out.println("TODO Implement!");
+		// TODO Implement staging the given states
+		// and persist that in a separate file
+		// Configuration.ensureLoaded();
+		// File latestReport = ReportFileUtils.getLatestReport();
+		// ChangeApplier applier = new ChangeApplier(checks);
+		// applier.apply(latestReport);
+	}
+
+	public static void main( String... args ) {
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/cli/subcommands/Stage.java]: AFAIK only the main entry point needs a `main`, which can be removed from all subcommands (see our CLI impl).

File: src/main/java/de/retest/recheck/cli/subcommands/Stage.java (hunk 2 above)
Comment (roesslerj) [reply src/main/java/de/retest/recheck/cli/subcommands/Stage.java]: How do you get the parameter description to show?

**Comment by roesslerj** [comment]
Done. Approve?

**Comment by beatngu13** [comment]
Gimme a second.

**Pull request merged**